app/code/Magento/ImportExport/Model/ResourceModel/Helper.php

```diff
@@ -53,11 +53,19 @@ public function getMaxDataSize()
     public function getNextAutoincrement($tableName)
     {
         $connection = $this->getConnection();
-        $entityStatus = $connection->showTableStatus($tableName);
-
-        if (empty($entityStatus['Auto_increment'])) {
+        $sql = sprintf(
+            'SELECT AUTO_INCREMENT FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME = %s AND TABLE_SCHEMA = DATABASE()',
+            $this->quote($tableName)
```

Comment:
This method doesn't exist. Here is test failure showing that:
```
1) Magento\CatalogImportExport\Model\Import\ImportWithNotExistImagesTest::testImportWithUnexistingImages
Error: Call to undefined method Magento\ImportExport\Model\ResourceModel\Helper::quote()

/var/www/html/app/code/Magento/ImportExport/Model/ResourceModel/Helper.php:58
/var/www/html/app/code/Magento/CatalogImportExport/Model/Import/Product/LinkProcessor.php:105
/var/www/html/app/code/Magento/CatalogImportExport/Model/Import/Product.php:1168
/var/www/html/app/code/Magento/CatalogImportExport/Model/Import/Product.php:1129
/var/www/html/app/code/Magento/ImportExport/Model/Import/Entity/AbstractEntity.php:628
/var/www/html/dev/tests/integration/tmp/sandbox--be468cf29f4a38f746cca31cdb1fc64149faf323e76cd34d89b850ef367fd698/generated/code/Magento/CatalogImportExport/Model/Import/Product/Interceptor.php:320
/var/www/html/dev/tests/integration/testsuite/Magento/CatalogImportExport/Model/Import/ImportWithNotExistImagesTest.php:198
/var/www/html/dev/tests/integration/testsuite/Magento/CatalogImportExport/Model/Import/ImportWithNotExistImagesTest.php:135
```

Could you fix that?